SQL/New_patches/2020-05-15-RemoveObsoleteMriViolationsLog.sql

```diff
@@ -0,0 +1,40 @@
+-- -----------------------------------------------------------------------------------
+-- When you set the Caveat flag of a scan to true in the imaging browser, a record
+-- is created in table mri_violations_log to indicate that there is a manual caveat 
+-- for that scan. If you subsequently set the Caveat back to false, this record is not
+-- deleted. This PR deletes those obsolete entries in table mri_violations_log
```

Comment:
Why would those be deleted? The `mrio_violations_log` is a log.

Reply:
I don't think this table is really a log per se. If you don't delete the violation associated to the Caveat, how are you going to know that there are no caveats anymore?

Reply:
By the caveat flag in the files table?